xray/fasthttp_test.go

```diff
@@ -0,0 +1,73 @@
+package xray
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/valyala/fasthttp"
+	"github.com/valyala/fasthttp/fasthttputil"
+)
+
+// serve serves http request using provided fasthttp handler
```

Comment:
Nit: serve is written twice.

Reply:
https://github.com/nicolascb/aws-xray-sdk-go/commit/f608cbf57ce01f4deda8d3614ddf66e66894afac